### ISSUE TITLE
Creation date format

### ISSUE THREAD

**Issue by MartijnMaandag** [joomla/joomla-cms#38552]
### Steps to reproduce the issue
I was testing the installation of the Dutch translation package for Joomla 4.2.1 (on a Joomla 4.2.0 site).
Some time ago the creation date in the Translation changed into YYYY-MM, nothing wrong with this, I use this in the translation.
Now I notice that the display in the extension list is changed from YYYY-MM (4.1.x) into YYYY-MM-DD (4.2.x).
This is confusing, if there are two releases in one month (4.2.x) they will have the same time-stamp (2022-08-01) and not 2022-08 like it was, which is less exact.
Attached the screenprints.

### Expected result
It is better to have YYYY-MM

### System information (as much as possible)

J4.2![screen shot 2022-08-22 at 10 41 40](https://issues.joomla.org/uploads/1/a0d3d0ae6a56cd3c135edc03b9f0dbfd.jpg)

### Additional comments

![screen shot 2022-08-22 at 10 41 40](https://issues.joomla.org/uploads/1/2727d4daf31895dcbe2800c9a502684a.jpg)

**Comment by richard67** [comment]
> This is confusing, if there are two releases in one month (4.2.x) they will have the same time-stamp (2022-08-01) and not 2022-08 like it was, which is less exact.

@MartijnMaandag I don't get that. The `YYYY-MM` format is also only accurate by the month. If you have a release at August 16 and then again at August 23, both will show up as `2022-08-01`. The day is added by the display, it seems, but in the XML file where this information comes from, we have only `YYYY-MM`. So I do not see how that should be more accurate.

**Comment by MartijnMaandag** [comment]
Well, 2022-08 is the right month and accurate, 2022-08-01 is not the right date and not accurate, this is confusing. I don't know why they changed this from 4.1 to 4.2.

**Comment by brianteeman** [comment]
https://github.com/joomla/joomla-cms/pull/27162

**Comment by Kostelano** [comment]
Yes, this is a bug. Moreover, it spread to all languages ​​(probably), and not just en-GB.

![Screenshot_1](https://user-images.githubusercontent.com/8440661/185979244-a859a582-2594-4f86-a766-1029c6dc3e94.png)

**Comment by brianteeman** [comment]
The pull request creating this was https://github.com/joomla/joomla-cms/pull/27162

There is a very long discussion there regarding this

**Comment by Kostelano** [comment]
The date format "does not know how" to read such a "truncated" date - for example, 2022-08. It converts it to [DATE_FORMAT_LC4](https://github.com/joomla/joomla-cms/pull/27162/files#diff-6a0163b4e4391b44d531ebd39c6d7085c55c9233472ee5d580e4a67dbfeab4f6R88).

For example, go to the `site/administrator/manifests/packages` directory and change the date in the `pkg_en-GB.xml` file to normal, then return to the **Extensions: Manage** and press the **Refresh Cache** button in the toolbar. The date will change back to normal.

**Comment by MartijnMaandag** [comment]
It is not only the language-packs, we could change them. All core extensions do have the same.

**Comment by brianteeman** [comment]
did you not read the discussion and pr I linked to #27162

**Comment by MartijnMaandag** [comment]
Yes, but my first statement that it is confusing is still there.
Why don't we change <creationDate>2022-08</creationDate into <creationDate>2022-08-23</creationDate in the languagepack? Then the actual date is shown and not the fake day 01.

**Comment by korenevskiy** [comment]
An XML file is a data file just like SQL is designed for data. The XML file is not a text document. We store data in SQL in ISO format. Maybe we will store data in SQL in the "Nov 2022" format?
